### PR TITLE
Fix timestamp offset

### PR DESCRIPTION
**Pull request by R-maan**

### CODE REVIEW

File: IonDotnet/Timestamp.cs [hunk 1]
```diff
@@ -361,7 +361,14 @@ private static int GetOffsetMinutes(string s, int startIdx)
                 throw new FormatException(s);
             }
 
-            return hour * 60 + minute;
+            int offset = hour * 60 + minute;
+
+            if (hour > 23 || minute > 59)
+            {
+                throw new FormatException($"{s} - Offset { offset } must be between -1439 and 1439 inclusive and minute between 00 and 59.");
```
Comment (byronlin13) [inline IonDotnet/Timestamp.cs]: Is the message "minute between 00 and 59" necessary? It is implied with the Offset must be between -1439 and 1439

Comment (R-maan) [reply IonDotnet/Timestamp.cs]: The reason I include that in the message is that, technically "2007-01-01T00:00-10:70" is between -1439 and 1439 but not valid.

Comment (R-maan) [reply IonDotnet/Timestamp.cs]: In amazon interface they throw this message in such cases: "Minute offset 70 above maximum or equal to : 60"
Yet I don't have a very strong opinion about it, I won't mind changing it if you think it's better.

Comment (byronlin13) [reply IonDotnet/Timestamp.cs]: I don't have strong opinion about this either. I think we can leave it as is.

**Comment by R-maan** [comment]
Updated the master, closing this one as it's already included.

**Pull request closed without merging**